components/backup-stream/src/metrics.rs

```diff
@@ -56,6 +56,11 @@ lazy_static! {
         exponential_buckets(1.0, 2.0, 16).unwrap()
     )
     .unwrap();
+    pub static ref INCREMENTAL_SCAN_DISK_READ: Counter = register_counter!(
+        "tikv_log_backup_initial_scan_disk_read",
```

Comment:
should change metrics too?

Reply:
Let's keep this series internal for now, 'cause we can get the rough throughput by the `IO breakdown` row.

Reply:
OK